src/main/java/com/rpg2014/Mapper/IronSpiderExceptionMapper.java

```diff
@@ -19,7 +21,13 @@ public Response toResponse(WebApplicationException e) {
         }if (e instanceof ForbiddenException){
```

Comment:
A switch statement for this would probably be more readable

Reply:
You cant switch on types in java.  I would have to compare a string of the type name, which is weird imo